### PR TITLE
Implement markup

### PR DESCRIPTION
**Pull request by yevheniiIvanise**
screenshot

### CODE REVIEW

File: scss/components/_services.scss [hunk 1]
```diff
@@ -1,13 +1,12 @@
 .services {
-  margin: 6.6rem 0 7.6rem;
-
   &__menu {
     display: flex;
     justify-content: space-between;
```
[the remark this answers is not in the document]
Comment (yevheniiIvanise) [reply scss/components/_services.scss]: Actually there is a problem with a container. Max-content width should be 590px as implied in the design. It is assumed to be nice when the container corresponds those 590px.

File: scss/components/_works.scss [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+    img {
+      width: 100%;
+    }
+  }
+
+  &__heading {
+    margin: 0 0 4.8rem;
+  }
+
+  &__paragraph {
```
Comment (ihor4568) [inline scss/components/_works.scss]: why empty selector here?

File: scss/components/_works.scss [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+  }
+
+  &__menu-item {
+    width: 18.1rem;
+  }
+
+  &__image {
+    margin-bottom: 4rem;
+
+    img {
```
Comment (ihor4568) [inline scss/components/_works.scss]: do not match on tags

File: scss/components/_services.scss [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+  }
+
+  &__menu-item {
+    width: 26.4rem;
+  }
+
+  &__image {
+    margin: 0 0 4.2rem;
+
+    img {
```
Comment (ihor4568) [inline scss/components/_services.scss]: do not match on tags

File: scss/components/_page.scss [hunk 5]
```diff
@@ -0,0 +1,11 @@
+.page {
+  font: $basic-font-size $basic-font-stack;
+
+  img {
```
Comment (ihor4568) [inline scss/components/_page.scss]: do not match on tags and do not use cascades - I mean `.page img` - You could possible have a class on an image, and in this case the selector would look like `.page__image`

File: scss/components/_information.scss [hunk 6]
```diff
@@ -0,0 +1,43 @@
...
+    margin: 6rem 0;
+  }
+
+  &__menu-item:last-child {
+    width: 29rem;
+    padding-left: 2rem;
+  }
+
+  &__image {
+    img {
```
Comment (ihor4568) [inline scss/components/_information.scss]: do not match on tags

File: scss/components/_header.scss [hunk 7]
```diff
@@ -0,0 +1,116 @@
...
+  &__subheading {
+    position: relative;
+    width: 48rem;
+    color: white;
+    font-size: 1.5rem;
+    text-transform: uppercase;
+    font-weight: bold;
+    margin: 0 auto 8rem;
+
+    &:before {
```
Comment (ihor4568) [inline scss/components/_header.scss]: looks like almost identical code for before and after. You could use `&:before, &:after { ... } ` to specify common styles, and then define different styles for each separately

File: scss/components/_footer.scss [hunk 8]
```diff
@@ -0,0 +1,25 @@
...
+
+  &__container {
+    padding: 2rem;
+  }
+
+  &__rights {
+    color: #aaacaf;
+    font-size: 1.4rem;
+
+    span {
```
Comment (ihor4568) [inline scss/components/_footer.scss]: do not match on tags

File: scss/components/_companies.scss [hunk 9]
```diff
@@ -0,0 +1,69 @@
...
+    font-size: 2.6rem;
+    font-family: Nexa;
+    opacity: 0.7;
+  }
+
+  &__menu-item:first-child {
+    font-family: Belinda;
+  }
+
+  &____menu-item:last-child {
```
Comment (ihor4568) [inline scss/components/_companies.scss]: 4 underlines? really?

File: index.html [hunk 10]
```diff
@@ -0,0 +1,253 @@
...
+          </div>
+        </li>
+        <li class="information__menu-item">
+          <h2 class="information__heading heading_2">just for you on
+            themeforest</h2>
+          <p class="information__paragraph paragraph">
+            Lorem ipsum dolor sit amet, consectetur adipiscing elit.
+            Donec interdum mollis risus, in mattis erat convallis egestas.
+          </p>
+          <button class="information__buton button grey" type="button" name="button">Read
```
Comment (ihor4568) [inline index.html]: is `grey` a separate block? It should probably be a modifier of `button ` or you could just describe this style in `information__buton `

File: scss/components/_button.scss [hunk 11]
```diff
@@ -0,0 +1,22 @@
...
+  &:hover {
+    cursor: pointer;
+    opacity: 0.9;
+  }
+
+  &:focus {
+    outline: none;
+  }
+
+  &.grey {
```
Comment (ihor4568) [inline scss/components/_button.scss]: do not use cascades - you can make this a modifier

File: index.html [hunk 12]
```diff
@@ -0,0 +1,253 @@
...
+      <h2 class="support__heading heading_2">fully supported template</h2>
+      <p class="support__paragraph paragraph">
+        Lorem ipsum dolor sit amet, consectetur adipiscing elit.
+        Suspendisse malesuada sodales risus, a adipiscing lorem commodo
+        vitae.
+        Ut semper semper lacus. Suspendisse convallis vel tellus at
+        consectetur.
+        Aenean risus arcu
+      </p>
+      <button class="services__read-more-button button grey" type="button" name="button">
```
Comment (ihor4568) [inline index.html]: is `grey` a separate block? It should probably be a modifier of button or you could just describe this style in services__read-more-button

File: package.json [hunk 13]
```diff
@@ -0,0 +1,42 @@
...
+    "gulp-sass": "4.0.1",
+    "husky": "^1.0.0-rc.14",
+    "js-beautify": "^1.8.6",
+    "lint-staged": "^7.2.2",
+    "node-sass": "4.9.3",
+    "prettier": "^1.14.3"
+  },
+  "husky": {
+    "hooks": {
+      "pre-commit": "lint-staged && npm run prettify-scss"
```
Comment (ihor4568) [inline package.json]: why do you run  `prettify-scss` after running lint-staged? The code should have already been formatted after `lint-staged` run.

File: package.json [hunk 14]
```diff
@@ -0,0 +1,42 @@
+{
+  "name": "task-01",
+  "version": "1.0.0",
+  "description": "",
+  "private": true,
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "start": "gulp",
+    "prettify-scss": "./node_modules/.bin/prettier --write *{/*.scss,/*/*.scss}"
```
Comment (ihor4568) [inline package.json]: there is no need to use full path here `./node_modules/.bin/`. You can use a command name directly in package.json  - I mean just `prettier --write .....`

**Comment by ihor4568** [comment]
Please attach screenshot of your markup

File: scss/components/_works.scss (hunk 2 above)
Comment (yevheniiIvanise) [reply scss/components/_works.scss]: fixed

File: index.html (hunk 10 above)
Comment (yevheniiIvanise) [reply index.html]: fixed

**Pull request merged**